src/internal/connector/exchange/service_iterators.go

```diff
@@ -113,9 +162,34 @@ func FilterContainersAndFillCollections(
 		}
 
 		edc.jobs = append(edc.jobs, jobs...)
+
+		if len(token) > 0 {
+			deltaTokens[*c.GetId()] = token
+		}
+	}
+
+	// TODO(ashmrtn): getFetchIDFunc functions should probably just return a
+	// multierror and all of the error handling should just use those so that it
+	// all ends up more consistent.
+	merrs := multierror.Append(nil, errs)
```

Comment:
While I agree with the sentiment, the most consistent thing to do here is continue to use `errs` and `support.WrapandAppend` instead of swapping to the multierr package.  Injecting these minor refactors piecemeal tends to accumulate chaff.  If we're concerned about error handling patterns, we can make an issue to change them in a consolidated effort.